gateway/api_v1.py

```diff
@@ -30,22 +30,29 @@ def logout():
     return {}, 201
 
 
-@app.route('/<path:varargs>', methods=['POST', 'GET'])
+@app.route('/')
+def index():
+    """List all known services."""
+    response = {
+        'services': list(configuration.bayesian_services.keys())
```

Comment:
Well you can sort the output of keys() so the tests will be stable.

Reply:
Thanks